README.md

```diff
@@ -170,7 +176,7 @@ npm run compile-exec
 Compile a specific package, e.g. the `.deb` for Debian:
 
 ```shell
-npm run release -- --dry --build-variant=Debian
+npm run evergreen-release package -- --dry --build-variant=Debian
```

Comment:
(This is for the changes in 57c1456fbc8dc9b9f71 and just a drive-by fix, in case it’s confusing that this is in here)